cmd/release/cmd/list.go

```diff
@@ -40,8 +41,38 @@ var rancherListRCDepsSubCmd = &cobra.Command{
 	},
 }
 
+var chartsListSubCmd = &cobra.Command{
+	Use:   "charts [branch] [charts](optional)",
+	Short: "List Charts assets versions state for release process",
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		var branch, chart string = "", ""
```

Comment:
Let's take advantage of the "0 value" and update this to be

```go

var branch string
var chart string
```

We don't need to explicitly set the new string to ""